exercises/Mathieu_Tillet/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,125 @@
+/**
+ * Skeleton main routine
+ */
+
+#include <iostream>
+#include <fstream>
+#include <map>
+#include <vector>
+#include <algorithm>
+#include <tuple>
+
+using namespace std;
```

Comment:
While it's fine for the exercise, out in the wild it is not a good idea to do this. In the environment where there are many libraries, this might cause conflicts. For example `shared_ptr` exists both in `std` and `boost`. So it is advised to not to use the whole namespace, but to specify explicitly `std` in front of each data structure or algorithm you are using.